tools/softwarecheck/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 3.10)
+project(softwarecheck)
+
+add_executable(${PROJECT_NAME} softwarecheck.cpp ${SOURCES})
+
+set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 17)
+
+set(DEPENDENCIES_DIR ../../dependencies/third-party/)
+set(SDK_DIR ../../sdk/include)
+
+include_directories(
+    ${DEPENDENCIES_DIR}
+    ${SDK_DIR}
+)
+
+set(SOURCES
+    ${DEPENDENCIES_DIR}/c_json/cJSON.c
```

Comment:
This works. But it's more flexible to use this:
target_link_libraries(${PROJECT_NAME} PRIVATE cjson)